### PR TITLE
[RA-1 Ch03]: 3.4.2.2. Network

### PR DESCRIPTION
**Pull request by ASawwaf**
remove "however, for small sites, a legacy topology (access/aggregation switches) can be set up."

### CODE REVIEW

**Comment by ASawwaf** [comment]
> @ASawwaf why 3.4.2.3 Storage has been removed by your proposed change? Your changes in lines 261 and 262 seem to be misconfigured and the new sentence does not make sense. Can you please correct it so we can review it?

@walterkozlowski , I don't why the storage clauses are removed ( wired ) , let me check 

thanks

**Comment by ASawwaf** [comment]
> > @ASawwaf why 3.4.2.3 Storage has been removed by your proposed change? Your changes in lines 261 and 262 seem to be misconfigured and the new sentence does not make sense. Can you please correct it so we can review it?
> 
> @walterkozlowski , I don't why the storage clauses are removed ( wired ) , let me check
> 
> thanks

Done

File: doc/ref_arch/openstack/chapters/chapter03.md [hunk 1]
```diff
@@ -268,6 +268,7 @@ Figure 3-3 shows a physical network layout where each physical server is dual ho
 [OpenStack](https://docs.openstack.org/arch-design/design-storage.html) supports many different storage architectures and backends. The choice of a particular backend storage is driven by a number of factors including: scalability, resiliency, availability, data durability, capacity and performance. 
 
 Most cloud storage architectures incorporate a number of clustered storage nodes that provide high bandwidth access to physical storage backends connected by high speed networks. The architecture consists of multiple storage controller units, each a generic server (CPU, Cache, storage), managing a number of high-performance hard drives. The distributed block storage software creates an abstract single pool of storage by aggregating all of the controller units. Advanced and high-speed networking (data routing) and global load balancing techniques ensure high-performance, high availability storage system
+ pool of storage by aggregating all of the controller units. Advanced and high-speed networking (data routing) and global load balancing techniques ensure high-performance, high availability storage system
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter03.md]: @ASawwaf this (line271) is duplicate of the last couple of lines from previous text (line 270). Also there is a typo in the previous text -- needs a "." after "... high availability storage system"

**Comment by pgoyal01** [comment]
Fixes #1311

**Pull request merged**